app/javascript/dashboard/routes/dashboard/conversation/ContactPanel.vue

```diff
@@ -137,7 +133,7 @@ import ContactDetailsItem from './ContactDetailsItem.vue';
 import ContactInfo from './contact/ContactInfo';
 import ConversationLabels from './labels/LabelBox.vue';
 import ContactCustomAttributes from './ContactCustomAttributes';
-import AvailabilityStatusBadge from 'dashboard/components/widgets/conversation/AvailabilityStatusBadge.vue';
+import multiselectDropdown from 'shared/components/ui/MutiselectDropdown.vue';
```

Comment:
Component name always should be in [**PascalCase**](https://vuejs.org/v2/style-guide/#Component-name-casing-in-JS-JSX-strongly-recommended)